exotica_core/include/exotica_core/kinematic_element.h

```diff
@@ -121,6 +121,8 @@ class KinematicElement
     KDL::Frame generated_offset = KDL::Frame::Identity();
     bool is_trajectory_generated = false;
     std::vector<double> joint_limits;
+    std::vector<double> velocity_limit;
```

Comment:
Thanks for renaming to singular. This should be a regular double and not a vector => it's a single value.